README.md

```diff
@@ -9,10 +9,16 @@ PyD provides seamless interoperability between Python and the D programming lang
 
 # Usage
 
-To use with dub, either specify the relevant subConfiguration, or run
-`source set_env_vars.sh <your python>` on linux or
-`set_env_vars.bat <your python>` on windows to set the relevant environment variables
-and use the `env` subConfiguration
+To use with dub, either specify the relevant subConfiguration for your python version,
```

Comment:
could you add this to somewhere suitable in the readthedocs as well? maybe create `docs/source/dub.rst`

Reply:
done

Reply:
actually, could you also add it to the list in index.rst?

Reply:
done